src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,80 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange = () => {},
+}: Props) => {
+  const lastPage = Math.ceil(total / perPage);
+
+  return (
+    <ul className="pagination">
+      <li
+        className={classNames('page-item', {
+          disabled: currentPage === 1,
+        })}
+      >
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          onClick={() => {
```

Comment:
Using `href="#prev"` can lead to unwanted side effects in the URL, which might not be desirable for a single-page application. Consider using `event.preventDefault()` in the `onClick` handler to prevent the default anchor behavior.